compiler/resolution/initializerResolution.cpp

```diff
@@ -333,6 +333,10 @@ void resolveNewInitializer(CallExpr* newExpr, Type* manager) {
   else if (manager == dtUnmanagedNilable)
     manager = dtUnmanaged;
 
+  if (manager == dtBorrowed) {
+    USR_FATAL(newExpr, "creating a 'new borrowed' type is a not allowed");
+  }
```

Comment:
The spec in rst/language/spec/classes.rst still describes `new borrowed C()` as equivalent to `(new owned C()).borrow()`.  This seems surprising to change without a release of deprecation.

Reply:
You're right. We're planning to deprecate it instead of it being an error before the upcoming release. But I first implemented it as an error and we had a deadline to get it done by yesterday so we temporarily went with the error.